src/metax_api/tests/api/rest/base/views/directories/read.py

```diff
@@ -599,8 +662,8 @@ def test_browsing_in_cr_context(self):
         '''
         cr_pk = CatalogRecord.objects.get(pk=1).identifier
         self._use_http_authorization(method='bearer', token=self.token)
-        response = self.client.get('/rest/directories/3/files?cr_identifier=%s' % cr_pk)
-        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        response = self.client.get('/rest/directories/3/files?cr_identifier={0}'.format(cr_pk))
```

Comment:
Try to refrain from random refactoring in feature PR's. The %-formatting operator is used in Metax a lot, a single refactor line here is not achieving anything.

Reply:
Agree. Probably idea was to do something else but didn't work out. 
Made both ct tests to look and act similar.